### PR TITLE
fix:docs capabilities

### PR DESCRIPTION
**Pull request by SivasubramanianV**
## Proposed changes

[//]: # (Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.)

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/main/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have added proper type definitions for new commands (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

**Comment by SivasubramanianV** [comment]
@christian-bromann Hi Christian, I have re-committed and opened new pull request. Please approve and merge.

File: website/docs/Capabilities.md [hunk 1]
```diff
@@ -46,12 +46,20 @@ While all driver offer different parameters for configuration, there are some co
 
 The path to the root of the cache directory. This directory is used to store all drivers that are downloaded when attempting to start a session.
 
-Type: `string`<br />
+Type: `string`
 Default: `process.env.WEBDRIVER_CACHE_DIR || os.tmpdir()`
 
+```sh
+
+```
```
Comment (christian-bromann) [inline website/docs/Capabilities.md:54]: like this

File: website/docs/Capabilities.md [hunk 2]
```diff
@@ -65,85 +73,97 @@ In order to propagate options to the driver you can use the following custom cap
...
-    {label: 'wdio:edgedriverOptions', value: 'msedge'},
-    {label: 'wdio:safaridriverOptions', value: 'safari'},
-  ]
+defaultValue="chrome"
+values={[
+{label: 'wdio:chromedriverOptions', value: 'chrome'},
+{label: 'wdio:geckodriverOptions', value: 'firefox'},
+{label: 'wdio:edgedriverOptions', value: 'msedge'},
+{label: 'wdio:safaridriverOptions', value: 'safari'},
+]
```
Comment (christian-bromann) [inline website/docs/Capabilities.md:82]: this

File: website/docs/Capabilities.md [hunk 3]
```diff
@@ -65,85 +73,97 @@ In order to propagate options to the driver you can use the following custom cap
...
 Custom bidi mapper path.
 
 Type: `string`
 
 ##### allowedIps
+
 Comma-separated allowlist of remote IP addresses which are allowed to connect to EdgeDriver.
 
-Type: `string[]`<br />
+Type: `string[]`
```
Comment (christian-bromann) [inline website/docs/Capabilities.md:159]: and these

**Review by christian-bromann — COMMENTED** [review]
Can you revert the changes that are not related to the original issue?

**Comment by SivasubramanianV** [comment]
@christian-bromann Sorry about that, I have not used git that much. I have over written my commit in previous pull request. Please check if that can be merged.

**Comment by christian-bromann** [comment]
> I have over written my commit in previous pull request. Please check if that can be merged.

Can you squash the commits in the other PR into one single commit to ensure the commit is made with the email you have used to sign the CLA?

**Pull request closed without merging**